rules/imports.js

```diff
@@ -0,0 +1,204 @@
+module.exports = {
+  env: {
+    es6: true,
+  },
+  parserOptions: {
+    ecmaVersion: 6,
+    sourceType: 'module',
+  },
+  plugins: ['import'],
+
+  settings: {
+    'import/resolver': {
+      node: {
+        extensions: ['.js', '.json'],
+      },
+    },
+    'import/extensions': ['.js'],
+    'import/core-modules': [],
+    'import/ignore': ['node_modules', '\\.(scss|css|less|hbs|svg|json)$'],
+  },
+
+  rules: {
+    // Static analysis:
+
+    // ensure imports point to files/modules that can be resolved
+    // https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/no-unresolved.md
+    'import/no-unresolved': ['error', {commonjs: true, caseSensitive: true}],
+
+    // ensure named imports coupled with named exports
+    // https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/named.md#when-not-to-use-it
+    'import/named': 'error',
+
+    // ensure default import coupled with default export
+    // https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/default.md#when-not-to-use-it
+    'import/default': 'off',
```

Comment:
Interestingly enough the [recommended](https://github.com/benmosher/eslint-plugin-import/blob/master/config/recommended.js) settings have this surface an error.  I guess that's a +1 for ES6 modules in Node.

Reply:
Yeah, I think erroring here is better in our case. Changing this.